src/main/scala/LongestPalindrome.scala

```diff
@@ -0,0 +1,44 @@
+object LongestPalindrome extends App {
```

Comment:
check whether your variables are getting value reassigned or not, as it might need to be val instead of val